templates/infinispan-ephemeral.json

```diff
@@ -200,29 +304,35 @@
                     "name": "hotrod",
                     "protocol": "TCP"
                   }
-                ]
+                ],
+                "volumeMounts": [
+                  {
+                    "name": "${APPLICATION_NAME}-configuration",
+                    "mountPath": "/opt/jboss/infinispan-server/standalone/configuration/custom"
+                  }
+                ],
+                "resources": {
+                  "requests": {
+                    "cpu": "0.5",
+                    "memory": "512Mi"
```

Comment:
For the OpenShift Online Staging environment (https://api.free-stg.openshift.com) there are the following limits defined:

```
$ oc describe limits                                                                                                                                             1 ↵
Name:			resource-limits
Namespace:		slaskawi
Type			Resource	Min	Max	Default Request	Default Limit	Max Limit/Request Ratio
----			--------	---	---	---------------	-------------	-----------------------
Pod			memory		150Mi	1Gi	-		-		-
Pod			cpu		29m	2	-		-		-
Container		cpu		29m	2	60m		1		-
Container		memory		150Mi	1Gi	307Mi		512Mi		-
PersistentVolumeClaim	storage		1Gi	1Gi	-		-		-
------------------------------------------------------------
```

So the request definitions fit into that:

```
$ oc describe statefulset infinispan-persistent-app
...
    Requests:
      cpu:	500m
      memory:	512Mi
...
```

The only problem that I'm struggling now is that I can't create the Role Binding resource (https://github.com/infinispan/infinispan-openshift-templates/pull/9/files#diff-63b24191146e34d876be199a9cec2ff0R28) which is necessary for clustering. But this is something I can discuss later (for Free Tier we are OK with having only 1 Pod).